lib/_http_incoming.js

```diff
@@ -349,6 +361,17 @@ IncomingMessage.prototype._dump = function _dump() {
   }
 };
 
+function onError(instance, cb, error) {
```

Comment:
Argument order like this is a bit non-standard for the Node.js ecosystem.
```suggestion
function onError(instance, error, cb) {
```

Reply:
Thank you, I'll change it.